infra/dockerfile.commit_retrieval

```diff
@@ -2,18 +2,21 @@ FROM mozilla/bugbug-base:latest
 
 COPY infra/git_hyper_blame_patch /root/git_hyper_blame_patch
 
-# Mercurial need Python2 :(
+# Setup mercurial from its own website, without install pip2 which has a lot of dependencies
+ARG MERCURIAL_VERSION="5.1"
+
 # git and git-hyper-blame (from depot_tools) are required by the annotate pipeline.
 RUN apt-get update && \
-    apt-get install -y python python-pip git curl && \
-    python2 -m pip install --disable-pip-version-check --no-cache-dir mercurial==5.1 && \
+    apt-get install --no-install-recommends -y python-minimal python-dev gcc bzip2 git curl && \
```

Comment:
Specify the 2 after python, so that if the default python becomes 3, we will still be installing 2.

Reply:
Ok, Will do that in the next edit.